ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -215,7 +215,7 @@ function suite() {
         TESTS="$(openshift-tests run --dry-run --provider "${TEST_PROVIDER}" "${TEST_SUITE}")"
         echo "${TESTS}" | grep -v "${TEST_SKIPS}" >/tmp/tests
         echo "Skipping tests:"
-        echo "${TESTS}" | grep "${TEST_SKIPS}"
+        echo "${TESTS}" | grep "${TEST_SKIPS}" || { exit_code=$?; echo 'Error: no tests were found matching the TEST_SKIPS regex:'; echo "$TEST_SKIPS"; return $exit_code }
```

Comment:
```
/bin/bash: -c: line 221: syntax error near unexpected token `fi'
```
in all jobs

Reply:
oh no :(

I have added the missing semicolon and the tests now look better.

Thanks for the /approve!